include/dca/phys/dca_step/cluster_solver/ctint/walker/ctint_walker_cpu_submatrix.hpp

```diff
@@ -652,13 +655,21 @@ auto CtintWalkerSubmatrixCpu<Parameters, Real, fix_rng_order>::computeAcceptance
   const int interaction_sign = configuration_.getSign(index_[0]);
   const int non_empty_sector = sector_indices_[0].size() ? 0 : 1;
 
+  Real mc_weight_ratio = acceptance_probability;
   Real K = total_interaction_;
+
   for (int v_id = 0; v_id < delta_vertices; ++v_id) {
     const auto field_type = configuration_.getSector(non_empty_sector)
                                 .getAuxFieldType(sector_indices_[non_empty_sector][v_id]);
     const auto b =
         configuration_.getSector(non_empty_sector).getLeftB(sector_indices_[non_empty_sector][v_id]);
-    K *= prob_const_[field_type][b] * interaction_sign;
+    K *= beta_ * prob_const_[field_type][b] * interaction_sign;
```

Comment:
I assume this is where the beta from above went to

Reply:
Yes, I moved it down as it is part of the probability but not the weight. The beta in the probability comes from the proposal probability.